SIRO_README.md

```diff
@@ -5,18 +5,18 @@ Project-specific README for SIRo.
 # Installation
 
 1. Clone the Habitat-lab [SIRo branch](https://github.com/facebookresearch/habitat-lab/tree/SIRo).
-1. Install Habitat-lab using [instructions](https://github.com/facebookresearch/habitat-lab/tree/SIRo#installation).
-1. Install Habitat-sim `main` branch.
+2. Install Habitat-lab using [instructions](https://github.com/facebookresearch/habitat-lab/tree/SIRo#installation).
```

Comment:
Markdown automatically numbers items in a number list when you view formatted markdown. It is easier to maintain if we just use all 1's instead of manually numbering items.

Reply:
Rolled back my changes